.packit.yaml

```diff
@@ -0,0 +1,86 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: datagrepper.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - datagrepper.spec
+    - .packit.yaml
+
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: datagrepper
+# downstream (Fedora) RPM package name
+downstream_package_name: datagrepper
+
+srpm_build_deps:
+  - poetry
+  - wget
+  - pytest
+
+actions:
+   create-archive:
+        - "poetry build -f sdist"
+        - "sh -c 'echo dist/datagrepper-$(poetry version -s).tar.gz'"
+   get-current-version:
+        - "poetry version -s"
+   post-upstream-clone:
+       # we need poetry v1.1.7 because of a bug in 1.1.8 where python evaluates incorrectly
+       - "sh -c 'wget https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py && python3 install-poetry.py --version 1.1.7'"
+       - "rm install-poetry.py"
+       # fetch specfile from src.fp.o
+       - "wget https://raw.githubusercontent.com/LenkaSeg/datagrepper/spec/datagrepper.spec -O datagrepper.spec"
```

Comment:
since you're actually including the spec file here, this can (and probably should) be dropped :)

just a note to everyone, when running `packit propose-downstream` on a commandline, make sure the version (the actual git ref) you are releasing has an up to date specfile. If not, you can force Packit to *not* check out the tag and use repository content from HEAD instead by specifying option `--local-content`

Reply:
Thanks! I dropped the spec wget.
Now I wonder why the rawhide rpmbuild and testing-farm are failing...that was already passing. F36 not yet, there the datanommer.models dependecy was not updated in bodhi yet.

Reply:
> Looks like you need to rebase :-)

rebased!